dom/events/Event-propagation.html

```diff
@@ -13,10 +13,10 @@
     var callback = function() { called = true };
     document.head.addEventListener("foo", callback);
     document.head.dispatchEvent(ev);
-    // Gecko resets the flags after dispatching; it will happily dispatch
+    // dispatchEvent resets the propagation flags so it will happily dispatch
     // the event the second time around.
     document.head.dispatchEvent(ev);
-    assert_equals(called, expected, "Propagation flag");
+    assert_equals(called, true, "Propagation flag");
```

Comment:
Shouldn't we separately test that the immediate propagation flag is reset?  (Or does this somehow test both independently?  I don't really understand event dispatch.)